azure-pipelines.yml

```diff
@@ -49,10 +48,16 @@ steps:
     checkLatest: true
 
 - task: PowerShell@2
-  displayName: 'Generate NuGet packages'
+  displayName: 'Generate Dawn NuGet packages'
   inputs:
     filePath: './Generate-NuGets.ps1'
-    arguments: '-Version $(GetAppVersionTask.appversion) -outputfolder $(nugetOutputPath)'
+    arguments: '-Version $(GetAppVersionTask.appversion) -outputfolder $(nugetOutputPath)' -flavour "Dawn"
+
+- task: PowerShell@2
+  displayName: 'Generate Browser NuGet packages'
+  inputs:
+    filePath: './Generate-NuGets.ps1'
+    arguments: '-Version $(GetAppVersionTask.appversion) -outputfolder $(nugetOutputPath) -flavour "Browser"
```

Comment:
Those "arguments" (on both tasks) need a single trailing comma, just before the flavour